tangos/tools/property_writer.py

```diff
@@ -278,7 +278,7 @@ def _needed_properties(self):
             needed.extend([name for name in x.requires_property()])
         return list(np.unique(needed))
 
-    def _needed_property_data(self):
+    def _required_property_names(self):
```

Comment:
Is this method ever used now? It feels like all uses cases are now encompassed in the previous method.